#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("distance_thr", double_t, 0, "", 0.05, 0.0, 1.0)
gen.add("buf_size", int_t, 0, "", 2, 0, 10)
exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "CloudOnPlane"))
